common/api-review/firestore.api.md

```diff
@@ -233,6 +233,9 @@ export function getFirestore(): Firestore;
 // @public
 export function getFirestore(app: FirebaseApp): Firestore;
 
+// @public
+export function getFirestore(): Firestore;
```

Comment:
Not sure if this is intentional, but `getFirestore(): Firestore` is already in this file on line 231

Reply:
Thanks for catching that, that looks like a bad merge in `packages/firestore/src/api/database.ts`. I kept the lower one because I don't really want the one with no args to be the overload that gets linked to (see doc fix in https://github.com/firebase/firebase-js-sdk/pull/6633).